### PR TITLE
Fix multiple typos [8]

### PR DESCRIPTION
**Pull request by JonHanna**

### CODE REVIEW

**Comment by JonHanna** [comment]
There were a few XML test cases where I wasn't sure if something was a deliberate error or not, so I left it, but with this batch, there's been a spell-check done on every solution in corefx except for expressions, because I got the idea for doing these after doing that project first, but I've that done in a branch off the branch for #7000, so its waiting on that.

File: src/System.Threading.Tasks/tests/Task/TaskContinueWithTests.cs [hunk 1]
```diff
@@ -1414,7 +1414,7 @@ private static Task<Int32> Choose(CancellationToken cancellationToken)
             //var boxedCompleted = new StrongBox<Task>(); // Acts as both completion marker and sync obj for targets
             var result = new TaskCompletionSource<int>();
 
-            // Set up teardown cancellation.  We will request cancellation when a) the supplied options token
+            // Set up tear-down cancellation.  We will request cancellation when a) the supplied options token
```
Comment (stephentoub) [inline src/System.Threading.Tasks/tests/Task/TaskContinueWithTests.cs]: Why is a dash being added here?

File: src/System.Xml.XDocument/tests/TreeManipulation/LoadFromReader.cs [hunk 2]
```diff
@@ -177,7 +177,7 @@ public void reader_14()
             ReadFrom(xml);
         }
 
-        //XElement: non-wellformed xml
+        //XElement: non�well-formed xml
```
Comment (stephentoub) [inline src/System.Xml.XDocument/tests/TreeManipulation/LoadFromReader.cs]: Should these be non-well-formed?

File: src/System.Xml.XDocument/tests/xNodeReader/ReadValue.cs [hunk 3]
```diff
@@ -75,7 +75,7 @@ public void TestReadValuePri0()
                         }
                     }
 
-                    TestLog.Compare(DataReader.ReadValueChunk(buffer, 0, 5), 5, "Didnt read 5 chars");
+                    TestLog.Compare(DataReader.ReadValueChunk(buffer, 0, 5), 5, "Dint read 5 chars");
```
Comment (stephentoub) [inline src/System.Xml.XDocument/tests/xNodeReader/ReadValue.cs]: All of these "Didnt"s should be "Didn't", not "Dint"

File: src/System.Threading.Tasks/tests/Task/TaskContinueWithTests.cs (hunk 1 above)
Comment (JonHanna) [inline src/System.Threading.Tasks/tests/Task/TaskContinueWithTests.cs]: Yes, that was overly picky of me; *teardown* is perfectly cromulent as the joke goes. I'll revert those.

**Comment by stephentoub** [comment]
LGTM other than the cited issues.

File: src/System.Xml.XDocument/tests/TreeManipulation/LoadFromReader.cs (hunk 2 above)
Comment (JonHanna) [inline src/System.Xml.XDocument/tests/TreeManipulation/LoadFromReader.cs]: I think I typed it as non–well-formed following the rule that an en-dash should be used when creating a compound from a compound, and it seems the dash got eaten in the save. A question therefore is, why didn't I get a warning on that?

**Comment by JonHanna** [comment]
I'm going to wait for CI to clear before fixing those, just in case a test is affected that I missed in local checks and I have to fix that too.

File: src/System.Xml.XDocument/tests/TreeManipulation/LoadFromReader.cs (hunk 2 above)
Comment (JonHanna) [inline src/System.Xml.XDocument/tests/TreeManipulation/LoadFromReader.cs]: Ah, it didn't get eaten, but it got saved encoded as something that this view isn't showing.

Comment (stephentoub) [inline src/System.Xml.XDocument/tests/TreeManipulation/LoadFromReader.cs]: We shouldn't change force an encoding of the file just to use the "right" dash in a comment.  In general we're trying to use only ASCII characters in the code files.

Comment (JonHanna) [inline src/System.Xml.XDocument/tests/TreeManipulation/LoadFromReader.cs]: ASCII-ified.

**Comment by JonHanna** [comment]
Test  Innerloop CentOS7.1 Debug Build and Test
Test Innerloop CentOS7.1 Release Build and Test 

(Hung Process)

**Comment by JonHanna** [comment]
Test Innerloop CentOS7.1 Debug Build and Test 
(Failed to mkdirs: /mnt/resource/j/workspace/dotnet_corefx/nativecomp_centos7.1_debug_prtest)

**Comment by stephentoub** [comment]
@mmitche, FYI on the mkdirs error

File: src/System.Xml.XDocument/tests/TreeManipulation/XContainerReplaceNodesOnXElement.cs [hunk 4]
```diff
@@ -11,11 +11,11 @@ public class XContainerReplaceNodesOnXElement : XContainerReplaceNodes
...
-            AddChild(new TestVariation(OnXElement) { Attribute = new VariationAttribute("XElement: Replace with single node") { Params = new object[] { 1, "<A xmlns='ns1' xmlns:p='nsp'><e1/>text1<p:e2>innertext<innerelem/></p:e2>text2<!--coment-->text3<?PI clicl?></A>", true }, Priority = 0 } });
-            AddChild(new TestVariation(OnXElement) { Attribute = new VariationAttribute("XElement: Replace with multiple nodes") { Params = new object[] { 4, "<A xmlns='ns1' xmlns:p='nsp'><e1/>text1<p:e2>innertext<innerelem/></p:e2>text2<!--coment-->text3<?PI clicl?></A>", true }, Priority = 1 } });
+            AddChild(new TestVariation(OnXElement) { Attribute = new VariationAttribute("XElement: Replace with single node") { Params = new object[] { 1, "<A xmlns='ns1' xmlns:p='nsp'><e1/>text1<p:e2>innertext<innerelem/></p:e2>text2<!--comet-->text3<?PI clicl?></A>", true }, Priority = 0 } });
+            AddChild(new TestVariation(OnXElement) { Attribute = new VariationAttribute("XElement: Replace with multiple nodes") { Params = new object[] { 4, "<A xmlns='ns1' xmlns:p='nsp'><e1/>text1<p:e2>innertext<innerelem/></p:e2>text2<!--comet-->text3<?PI clicl?></A>", true }, Priority = 1 } });
```
Comment (stephentoub) [inline src/System.Xml.XDocument/tests/TreeManipulation/XContainerReplaceNodesOnXElement.cs]: These should all be comment, not comet.

File: src/System.Xml.XDocument/tests/TreeManipulation/XNodeReplaceOnElement.cs [hunk 5]
```diff
@@ -10,16 +10,16 @@ public class XNodeReplaceOnElement : XNodeReplace
...
+            AddChild(new TestVariation(OnXElement) { Attribute = new VariationAttribute("(BVT)XElement (in document): Replace with multiple nodes") { Params = new object[] { 2, "<A xmlns='ns1' xmlns:p='nsp'><e1/>text1<p:e2>innertext<innerelem/></p:e2>text2<!--comet-->text3<?PI clicl?></A>", false }, Priority = 0 } });
             AddChild(new TestVariation(OnXElement) { Attribute = new VariationAttribute("XElement: Replace with multiple nodes - one child") { Params = new object[] { 4, "<A><?Pi data?></A>" }, Priority = 2 } });
-            AddChild(new TestVariation(OnXElement) { Attribute = new VariationAttribute("XElement (in document): Replace with multiple nodes") { Params = new object[] { 4, "<A xmlns='ns1' xmlns:p='nsp'><e1/>text1<p:e2>innertext<innerelem/></p:e2>text2<!--coment-->text3<?PI clicl?></A>", false }, Priority = 1 } });
+            AddChild(new TestVariation(OnXElement) { Attribute = new VariationAttribute("XElement (in document): Replace with multiple nodes") { Params = new object[] { 4, "<A xmlns='ns1' xmlns:p='nsp'><e1/>text1<p:e2>innertext<innerelem/></p:e2>text2<!--comet-->text3<?PI clicl?></A>", false }, Priority = 1 } });
```
Comment (stephentoub) [inline src/System.Xml.XDocument/tests/TreeManipulation/XNodeReplaceOnElement.cs]: comment, not comet

File: src/System.Xml.XDocument/tests/xNodeBuilder/FunctionalTests.cs [hunk 6]
```diff
@@ -921,7 +921,7 @@ public override void AddChildren()
...
                     this.AddChild(new TestVariation(testIntoOpenedWriterDefaultNS) { Attribute = new VariationAttribute("Write into used reader II. (def. ns.)") { Params = new object[] { "<A xmlns='ns-other'><B xmlns='nsp'><C xmlns='nsp'/></B></A>", "<root xmlns='nsp'><A xmlns='ns-other'><B xmlns='nsp'><C/></B></A></root>" }, Priority = 2 } });
-                    this.AddChild(new TestVariation(testIntoOpenedWriterXlinqLookup1) { Attribute = new VariationAttribute("Write into used reader (Xlinq lookup + existing hint in the Writer; different prefix)") { Params = new object[] { "<p1:root xmlns:p1='nsp'><p2:B xmlns:p2='nsp'/></p1:root>" }, Priority = 2 } });
+                    this.AddChild(new TestVariation(testIntoOpenedWriterXlinqLookup1) { Attribute = new VariationAttribute("Write into used reader (Linq lookup + existing hint in the Writer; different prefix)") { Params = new object[] { "<p1:root xmlns:p1='nsp'><p2:B xmlns:p2='nsp'/></p1:root>" }, Priority = 2 } });
```
Comment (stephentoub) [inline src/System.Xml.XDocument/tests/xNodeBuilder/FunctionalTests.cs]: Why did this change from xlinq to linq?

**Comment by JonHanna** [comment]
I'm going to re-do this entire batch again. Some "replace alls" were clearly not correct, so I need to review it myself, never mind submit it for review.

**Pull request closed without merging**